### DISCUSSION TITLE
Protected site

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#2003] [Q&A]
Hello

I want to put a password to the site.

I have implemented the following in the deploy-to-staging.yml file

![image](https://github.com/open-sdg/open-sdg/assets/117297044/241c78cc-043a-4bd3-8fa3-2fe023b6c9e5)

Before the platform update it worked, but not anymore.

Why this situation?

Thank you

**Comment by brockfanning** [comment]
@SandraEliz I'm not sure what could have changed that would have affected that kind of thing. That appears to be something that runs after the site has already been built. So I don't think anything that would have changed in a new version of the platform would make a difference. However, is it possible that the build itself is failing for some other reason? Do you see any error messages?

**Reply by SandraMarq** [reply]
Hello
This is what appears

![image](https://github.com/open-sdg/open-sdg/assets/118024145/d58c0af7-da8c-499c-9371-e67fa666652f)

Is there a way to add a login page with a password?

**Reply by brockfanning** [reply]
@SandraEliz I've never tried the password protection so I'm not much help there. But just to rule out other issues, you might try clearing your browser cache, or trying a different browser that you don't normally use. Also, on the off chance you are using a government-issued computer, you might try on your personal computer instead. These are all just "shots in the dark", hopefully someone else has more experience with this.
